### ISSUE TITLE
migration runs twice in parallel processes

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#408]
I thought I used a lock to prevent this behaviour! That's dangerous!

**Comment by urapadmin** [comment]
it all looks okay and I cannot reproduce it. But I have set the lifetime for the lock from 60seconds to 120 seconds. Perhaps that was problematic?

**Comment by urapadmin** [comment]
should be evaluated again when caring about standalone task management

**Comment by urapadmin** [comment]
I could build it into the MCP program one day, but starting it simply as an MCP job is unnecessary. Postponed.

**Comment by urapadmin** [comment]
see #495

**Comment by urapadmin** [comment]
in fact it would be better if migration did not run at all unless absolutely necessary on kiosk startup. see #1440.
Of course, it would still have to run after a restore, but only then.

**Comment by urapadmin** [comment]
this is pretty much a duplicate of #495 and that's closed. Migration should not be running parallel anywhere.
